pkgs/by-name/br/breakpointHook/attach.sh

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env bash
+
+id="$1"
+pid="$(pgrep -f "sleep $id" | head -n1)"
```

Comment:
If no process matches this or multiple processes matches this, it should become an error.